docs/reference/msbuild-targets.md

```diff
@@ -336,9 +336,14 @@ Additional restore settings may come from MSBuild properties in the project file
 | RestoreConfigFile | Path to a `Nuget.Config` file to apply. |
 | RestoreNoCache | If true, avoids using cached packages. See [Managing the global packages and cache folders](../consume-packages/managing-the-global-packages-and-cache-folders.md). |
 | RestoreIgnoreFailedSources | If true, ignores failing or missing package sources. |
+| RestoreFallbackFolders | Fallback folders, used in the same way the user packages folder is used. |
+| RestoreAdditionalProjectSources | Additional sources to the resulting list of sources |
+| RestoreAdditionalProjectFallbackFolders | Additional fallback folders appended to the resulting list of fallback folders |
```

Comment:
Additional fallback folders?
Additional fallback folders to use during restore?